CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ See the [public meeting
 notes](https://docs.google.com/document/d/1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glWdHj8/edit?usp=sharing)
 for a summary description of past meetings. To request edit access, join the
 meeting or get in touch on
-[Gitter](https://gitter.im/open-telemetry/opentelemetry-dotnet).
+[Slack](https://cloud-native.slack.com/archives/C01N3BC2W7Q).
```

Comment:
Do we need to keep a link to old gitter for anyone interested in searching the history?

Reply:
It is in the commit history if folks want to dig further.
I guess we want to hide it since we want to deprecate gitter (and potentially remove it)?